shell/assets/styles/themes/_light.scss

```diff
@@ -26,7 +26,7 @@ $disabled  : $medium;
 $primary         : #3D98D3;
 $secondary       : $darker;
 $link            : #3D98D3;
-$keyboard-focus  : #3300ff;
+$keyboard-focus  : #{darken($primary, 10%)};
```

Comment:
This is the same as `primary-hover-bg`, which is the variable used for the mouse hover color in buttons